### ISSUE TITLE
Possible packaging problem with zip and title case

### ISSUE THREAD

**Issue by Marxsal** [kookma/TW-Gatha#2]
I have a plugin like: $:/plugins/MAS/DynaNotator

When I exported this via zip, all of the tiddler files went into

> mas/

But the tiddlywiki.info file went into

> MAS/

On Windows these two might have been combined, but on Linux (and Mac, no doubt) they are two separate things.

Gatha 0.9.0

**Comment by kookma** [comment]
Hi Mark,
I investigate to see what is the problem. I am not sure if I used any name conversion (like convert to lowecase or not).
I will look in to in this weekend.

**Comment by Marxsal** [comment]
Many plugins use lowercase only. But I don't know if that is a rule.

**Comment by kookma** [comment]
> Many plugins use lowercase only. But I don't know if that is a rule.

I do not think it is a rule! One question: the tiddler name seems works fine, the created plugin also works fine, but I see you like to have the same folder, file name ( with the same uppercase/lowercase)

**Comment by kookma** [comment]
Hi Mark,
 I went into the code, the reason is using slugify operator. It converts all uppercase into lower case.
So, may be Jeremy can explain what is the reason to use lowercase when slugify!
